DgmlPowerTools.2022/DgmlPowerTools.2022/Utilities/ProjectDependencies.cs

```diff
@@ -82,7 +84,7 @@ public Graph GetProjectDependencies(IVsSolution solution)
                         GraphCategory projectCategory = GetOrCreateProjectCategoryStyle(graph, custom, kind);
                         GraphNode sourceProjectNode = graph.Nodes.GetOrCreate(id, label, projectCategory);
 
-                        CreateNugetReferences(sourceProject.FullName, graph, sourceProjectNode, versionProperty);
```

Comment:
using the 'versionProperty' was just a tiny bit more efficient, it saves the GraphModel from having to go find the GraphProperty in the various Schemas associated with the graph, so I'd prefer to keep that if possible.